docs/data_usage_guides/food_security_and_nutrition.md

```diff
@@ -31,10 +31,15 @@ For available query parameters, please see the
   and applying the algorithm from
   [`hdx-python-country`](https://hdx-python-country.readthedocs.io/en/latest/),
   which uses phonetic name matching and manual overrides
-* Any rows whose p-codes are unmatched are p-coded at the national level.
+* Where admin 1 names could not be p-coded, the provided p-codes from the 
+  source data are at national level
+* Where admin 2 names could not be p-coded, the provided p-codes from the 
+  source data are at admin 1 level if possible or national level if not
 
 ### Usage Notes
 
+* The data is available at national, admin 1 and admin 2 with admin names 
+  supplied in the returned data along with p-codes where available
```

Comment:
which admin names supplied do you mean - should we say, both provider and standardized?